code/src/App.js

```diff
@@ -1,29 +1,37 @@
 import React from "react";
-import { BarcodeScanner } from "components/BarcodeScanner";
+import { combineReducers, configureStore } from "@reduxjs/toolkit";
+import { productsReducer } from "reducers/productsReducer"
+import { Provider } from "react-redux"
+import { ScanBarcode } from "components/ScanBarcode.js";
+import styled from 'styled-components';
 
-const onDetected = (code) => {
-  console.log(`Code: ${code}`);
-  fetch(`https://world.openfoodfacts.org/api/v0/product/${code}.json`)
-    .then((data) => data.json())
-    .then((json) => {
-      console.log(json);
-    });
-};
+const reducer = combineReducers({ productsReducer: productsReducer.reducer })
+
+export const store = configureStore({ reducer })
 
 export const App = () => {
   return (
-    <div>
-      <label>
-        {" "}
-        Test codes here:{" "}
-        <input type="text" onChange={(e) => onDetected(e.target.value)}></input>
-      </label>
-      <p>
-        {" "}
-        Use the field above to test barcodes manually and keep an eye on your
-        console in the browser. i.e. Type 7311070347272 - Pågen Gifflar. Yum
-      </p>
-      <BarcodeScanner onDetected={onDetected}></BarcodeScanner>
-    </div>
+    <Provider store={store}>
+      <Container>
+        <Title>How much?</Title>
+        <Paragraph>Scan a product and compare values to daily recommendations</Paragraph>
+        <ScanBarcode />
+      </Container>
+    </Provider>
   );
 };
+
+const Container = styled.main`
+  text-align: center;
+  margin: 40px;
+`
+const Title = styled.h1`
+  font-family: 'Roboto', sans-serif;
+  font-size: 60px;
+  color: #ffa6a6;
+  text-shadow: 2px 2px #FF0000;
+  margin: 0;
+`
+const Paragraph = styled.p`
+  margin: 0;
+`
```

Comment:
Don't forget that variables should be declared before they are used - meaning - define your styled components above App.js instead of below.